astropy/coordinates/representation/cylindrical.py

```diff
@@ -134,6 +134,22 @@ def _scale_operation(self, op, *args):
             result.differentials[key] = differential.__class__(*new_comps, copy=False)
         return result
 
+    def represent_as(self, other_class, differential_class=None):
+        if isinstance(other_class, type):
+            if issubclass(other_class, PhysicsSphericalRepresentation):
+                diffs = self._re_represent_differentials(
+                    other_class, differential_class
+                )
+                r = np.sqrt(self.rho**2 + self.z**2)
```

Comment:
Wouldn't it be better to use [`np.hypot()`](https://numpy.org/doc/stable/reference/generated/numpy.hypot.html)?

Reply:
`np.hypot` indeed takes proper care when one or the other is small.

Reply:
✅